Samples/Sample XML/StudentEnrollment.xml

```diff
@@ -14,7 +14,7 @@
 		<EntryDate>2021-08-23</EntryDate>
 		<EntryGradeLevel>uri://ed-fi.org/GradeLevelDescriptor#Fifth grade</EntryGradeLevel>
 		<GraduationPlanReference ref="GPLN_Minimum-GBES"/>
-		<CalendarReference>
+ 		<CalendarReference>
```

Comment:
```suggestion
		<CalendarReference>
```

Reply:
This was just a mistake and I restored the <CalendarReference> and file validates.

Reply:
This is still showing as extra white space being added in GitHub.